File: frontends/go-sh/testdata/t60_contains.go
// t60_contains: substring containment
// diagnostics: program prints its result to stdout
s := "hello world"
if strings.Contains(s, "world") {
    fmt.Println("has")
} else {
    fmt.Println("no")
}
if strings.Contains(s, "nope") {
    fmt.Println("has2")
} else {
    fmt.Println("no2")
}

File: frontends/perl-sh-go/testdata/t60_contains.pl
# t60_contains: substring containment
# diagnostics: program prints its result to stdout
my $s = "hello world";
print index($s, "world") >= 0 ? "has\n" : "no\n";
print index($s, "nope") >= 0 ? "has2\n" : "no2\n";
